src/mixins/canvas_events.mixin.js

```diff
@@ -268,10 +352,11 @@
      * @param {Event} e Event object fired on mouseup
      */
     _onMouseUp: function (e) {
+      this._removePointer(e);
       this.__onMouseUp(e);
       this._resetTransformEventData();
       var canvasElement = this.upperCanvasEl,
-          eventTypePrefix = this.enablePointerEvents ? 'pointer' : 'mouse';
+          eventTypePrefix = this._getEventPrefix();
```

Comment:
we want to do the next 15 lines of code, only when we are finished with pointers, or maybe JUST for mouse events, since pointer events can carry this information anyway in the touches / changedPointers ( or whatever they have ).